packages/react-shopify-app-bridge/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gadgetinc/react-shopify-app-bridge",
-  "version": "0.10.2",
+  "version": "1.10.2",
```

Comment:
I would probably just bump this to `0.11.2`. Minor versions > 1.0.0 are pretty much the same as major version changes. Were you able to test this locally? I think I had tried to do this at some point as was running into some problems with Shopify embedded apps.